manim/mobject/geometry/labeled.py

```diff
@@ -67,17 +72,17 @@ def __init__(
         font_size: float = DEFAULT_FONT_SIZE,
         label_color: ParsableManimColor = WHITE,
         label_frame: bool = True,
-        frame_fill_color: ParsableManimColor = None,
+        frame_fill_color: ParsableManimColor | None = None,
         frame_fill_opacity: float = 1,
-        *args,
-        **kwargs,
+        *args: Any,
+        **kwargs: Any,
     ) -> None:
         label_color = ManimColor(label_color)
         frame_fill_color = ManimColor(frame_fill_color)
         if isinstance(label, str):
             from manim import MathTex
 
-            rendered_label = MathTex(label, color=label_color, font_size=font_size)
+            rendered_label = MathTex(label, color=label_color, font_size=font_size)  # type: Union[Tex, MathTex, Text]
```

Comment:
I would prefer not to use `# type:` comments, `rendered_label: Tex | MathTex | Text` would be better.
Also, is typing this necessary? I thought mypy might be able to do this on it's own.

Reply:
It is necessary to reasons similar to an earlier issue https://github.com/ManimCommunity/manim/pull/3961#discussion_r1816097638

I will move the type comments as suggested.